man/systemd.exec.xml

```diff
@@ -132,8 +132,19 @@
         the <function>chroot()</function> jail. Note that setting this parameter might result in additional
         dependencies to be added to the unit (see above).</para>
 
-        <para>The <varname>PrivateUsers=</varname> setting is particularly useful in conjunction with
-        <varname>RootDirectory=</varname>. For details, see below.</para></listitem>
+        <para>The <varname>MountAPIVFS=</varname> and <varname>PrivateUsers=</varname> settings are particularly useful
+        in conjunction with <varname>RootDirectory=</varname>. For details, see below.</para></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>MountAPIVFS=</varname></term>
+
+        <listitem><para>Takes a boolean argument. If on, a private mount namespace for the unit's processes is created
+        and the API file systems <filename>/proc</filename>, <filename>/sys</filename> and <filename>/dev</filename>
+        will be mounted inside of it, unless they are already mounted. Note that this option has no effect unless used
+        in conjunction with <varname>RootDirectory=</varname> as these three mounts are generally mounted in the host
+        anyway, and unless the root directory is changed the private mount namespace will be a 1:1 copy of the host's,
+        and include these three mounts.</para></listitem>
```

Comment:
If I'm reading the code correctly, `/dev/` is bind mounted from the host. I think that this is worth mentioning explicity. In particular, using `PrivateDevices` might be desirable for services using this option.